examples/csaf/README.md

```diff
@@ -0,0 +1,37 @@
+# CSAF Examples
+
+This directory is intended to contain CSAF files to test the OpenVEX tooling
+capabilities to read CSAF VEX. On the long run, as the feature matures, we
+should write some tests to ensure we are ingesting the properly.
```

Comment:
nit:

```suggestion
should write some tests to ensure we are ingesting CSAF properly.
```

Reply:
I've added an expanded example to the CSAF readme and fixed the nit above